src/translations/users.translations.js

```diff
@@ -0,0 +1,4 @@
+export const userTranslations = {
```

Comment:
We don't need `translations` folder, it will confuse developers a little bit because we don't translate anything it is better to rename it to the `users.messages` because it just reflects messages

Reply:
removed to config.js file, error status object